lib/taiko.js

```diff
@@ -1669,11 +1672,16 @@ async function mouseAction(
   validate();
   var actions = ['press', 'move', 'release'];
   if (
-    (isSelector(selector) ||
-      isString(selector) ||
-      Number.isInteger(selector)) &&
-    !actions.includes(selector)
+    !actions.includes(selector) &&
+    !isSelector(selector) &&
+    !isString(selector)
   ) {
+    throw Error(
+      'Kindly pass valid selector/string, Invalid Selector value passed : ' +
```

Comment:
please remove "Kindly pass valid selector/string," 

The first few characters of error message are most important, and it helps to keep them crisp.